pkg/app/pipedv1/plugin/kubernetes_multicluster/deployment/pipeline.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2025 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deployment
+
+import (
+	"slices"
+
+	"github.com/pipe-cd/pipecd/pkg/plugin/sdk"
+)
+
+const (
+	// StageK8sSync represents the state where
+	// all resources should be synced with the Git state.
+	StageK8sMultiSync string = "K8S_MULTI_SYNC"
+	// StageK8sRollback represents the state where all deployed resources should be rollbacked.
+	StageK8sMultiRollback string = "K8S_MULTI_ROLLBACK"
```

Comment:
It might be forgotten to change the comment.

```suggestion
	// StageK8sMultiSync represents the state where
	// all resources should be synced with the Git state.
	StageK8sMultiSync string = "K8S_MULTI_SYNC"
	// StageK8sMultiRollback represents the state where all deployed resources should be rollbacked.
	StageK8sMultiRollback string = "K8S_MULTI_ROLLBACK"
```